libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,7 +1,7 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
     <div>
-      <img class="reading-list-item--cover" [src]="b.coverUrl" />
+      <img class="reading-list-item--cover" [src]="b.coverUrl" [alt]="b.title + ' image'" />
```

Comment:
Since the cover image doesn't convey useful non-visual information it is better to use  `alt=""`.

Reply:
Ahh I guess the role would need to change here to something like `role="presentation"`